docs/builder_author_faq.md

```diff
@@ -0,0 +1,18 @@
+## When should a Builder build to `cache` vs `source`?
+
+Use `build_to: source` when:
+- The generated code _does not_ depend on details outside of the package with
```

Comment:
I think I understand this first sentence but it is very hard for me to parse. Can you reword it? The multiple references to generated code specifically cause the confusion. Maybe use "input sources" for the second reference?

Reply:
Rephrased. Is it more clean now?